.travis.yml

```diff
@@ -45,7 +46,10 @@ before_script:
 
 script:
     - make
-    - if [ -z "$CROSS_COMPILE" ]; then make test; fi
+    - if [ -n "$CROSS_COMPILE" ]; then
+          export EXE_PRE="wine " EXE_EXT=".exe" WINEPREFIX=`pwd`;
```

Comment:
Can you test following? In util/shlib_wrap.sh you'll find `cmd="$1${EXE_EXT}"`. Replace it with ` cmd="$1"; [ -f $cmd ] || cmd="$cmd${EXE_EXT}"`. Question is if make test makes any progress.